spirv_msl.cpp

```diff
@@ -1319,7 +1331,7 @@ void CompilerMSL::extract_global_variables_from_function(uint32_t func_id, std::
 				{
 					if (added_in)
 						continue;
-					name = input_wg_var_name;
```

Comment:
Why this change?

Reply:
Because in the new implementation, the parameter name to the entry point is no longer the same as the name that gets used in the generated shader code.

Reply:
Looking at this code again, I don't understand this. It seems like in other places, input_wg_var_name is not used either. Is this variable even necessary? Is there a reason for it to be != "gl_in"?